lib/Twig/Extensions/Node/Trans.php

```diff
@@ -45,11 +44,14 @@ public function compile(Twig_Compiler $compiler)
             $message = trim($notes->getAttribute('data'));
 
             // line breaks are not allowed cause we want a single line comment
-            $message = str_replace(array("\n", "\r"), " ", $message);
+            $message = str_replace(array("\n", "\r"), ' ', $message);
             $compiler->write("// notes: {$message}\n");
         }
 
-        if ($vars) {
+        if (
+            $vars 
+            or null !== $withVars
```

Comment:
should be ``||``, not ``or`` for the boolean operator.

and I would keep it on 1 line here as the condition stays short enough